### ISSUE TITLE
Reproduce the code in the paper, test the performance, and find that the performance is much worse than the performance claimed in the paper. The performance in Univariate TSC dataset

### ISSUE THREAD

**Issue by baobingzhang** [QiaoXiao7282/DSN#3]
When I tried to run the code and reproduce the paper, I encountered the same problems as the previous two issues. I fixed these problems myself, but when I tried to test the UCR_85 dataset myself, the results were very poor. Almost all of the 85 datasets were much worse than the results stated in the paper. I want to know why? And I didn’t find any CAM related code.

**Comment by QiaoXiao7282** [comment]
Thank you for your interest in our work. Regarding your training, did the training loss or accuracy converge to satisfactory levels? Also, is the PyTorch version you are using the same as ours?
